README.md

```diff
@@ -1,2 +1,37 @@
 # programapi
-Program API
+
+This project downloads, processes, saves, and serves the static JSON files containing details of accepted speakers and submissions via an API.
+
+What this project does step-by-step:
+
+1. Downloads the Pretalx speaker and submission data, and saves it as JSON files.
+2. Transforms the JSON files into a format that is easier to work with and OK to serve publicly. This includes removing unnecessary/private fields, and adding new fields.
+3. Serves the JSON files via an API.
+
+## Installation
+
+1. Clone the repository.
+2. Install the dependency management tool: ``make deps/pre``
```

Comment:
Maybe an idea for the future. What about we just install `programapi` as a Python package? And have a following functionality:
* `programapi download`
* `programapi transform`
By the way do you think that we can avoid saving unnecessary / private fields when downloading?

Reply:
I thought about it and didn't want to do it when there was bunch of other stuff to implement first. But I support it, I think we can do it after this PR.

About the private/unused fields:

- IINM some fields like email, ``do_not_record`` cannot be excluded when downloading. 
- Answers can probably be excluded, but we do ``?questions=all`` to get all the answers, so we can manage what to include/exclude in the model ad-hoc.